### ISSUE TITLE
Need help

### ISSUE THREAD

**Issue by Andrew-Zn** [projectstorm/react-diagrams#228]
I would like to ask a question in my react projects refer to the library, but I in document examples drawn only three fixed length of the cuboid, width is a screen length, also can't drag!
Did I miss anything?

**Comment by Andrew-Zn** [comment]
Run the example, can not display correctly! I am a foreign developer, English is not very good, the document is too careles

**Comment by Andrew-Zn** [comment]
I am a foreign developer, English is not very good, who can instruct me, I will be extremely grateful, the document is too simple.

**Comment by ajthinking** [comment]
Can you rephrase "in document examples drawn only three fixed length of the cuboid" or attach an image of your problem?

**Comment by Andrew-Zn** [comment]
I'll just copy the example in the demo and run it and get the result

![1111](https://user-images.githubusercontent.com/31008185/38844539-84b902e0-4226-11e8-8b69-4356d608aa01.png)

**Comment by ajthinking** [comment]
This is most likely a css issue. Can you confirm you have loaded the css properly?

In my case:
```
@import "~storm-react-diagrams/src/sass/main";
```

**Comment by Andrew-Zn** [comment]
The introduction of CSS would be a blank, and I deleted the "position=relative" under the ".srd-diagram", and the result was this.
![222](https://user-images.githubusercontent.com/31008185/38851384-1ed96bd2-4248-11e8-8bed-26de18b5b4fe.png)

**Comment by ajthinking** [comment]
Ok, that looks good? Not sure about this, but you might want to check your parent div if it got any interfering properties. Good luck!

**Comment by Andrew-Zn** [comment]
Thank you for your help.

**Comment by ganesh-sankey** [comment]
@Andrew-Zn You need to import style.min.css into your parent div/component then it will work.
import 'storm-react-diagrams/dist/style.min.css';

**Comment by Andrew-Zn** [comment]
Yes, I did, but the problem is still there, I have to delete it
![image](https://user-images.githubusercontent.com/31008185/39038077-206b5c2a-44b5-11e8-89d2-31c5350b139d.png)

**Comment by Andrew-Zn** [comment]
CSS has been correctly referenced. It may take some more time for me to find where the problem is.I will reply to you as soon as there is new progress. Thank you for your patient guidance.

**Comment by Andrew-Zn** [comment]
It`s Ok,Add a height to the srd-diagram.

**Comment by sjy** [comment]
OMO its better to set a default height rule to .src-diagram
